docs/02-developers/04-quickstart/remix.md

```diff
@@ -0,0 +1,180 @@
+4. ABI encoded arguments: if the arguments provided are ABI encoded, check this option.
+
+> Copy only the information inside the input code block {}, paste the code into a file named `standard-json-input.json` and click on add file:
+
+![Input Block - Rootstock Testnet Explorer](/img/developers/quickstart/13-remix.png)
+
+Here’s the example code block within the curly brackets:
+
+```solidity
+ "language": "Solidity",
+       "sources": {
+           "contracts/Storage.sol": {
+               "content": "// SPDX-License-Identifier: GPL-3.0\n\npragma solidity >=0.8.2 <0.9.0;\n\n/**\n * @title Storage\n * @dev Store & retrieve value in a variable\n * @custom:dev-run-script ./scripts/deploy_with_ethers.ts\n */\ncontract Storage {\n\n    uint256 number;\n\n    /**\n     * @dev Store value in variable\n     * @param num value to store\n     */\n    function store(uint256 num) public {\n        number = num;\n    }\n\n    /**\n     * @dev Return value \n     * @return value of 'number'\n     */\n    function retrieve() public view returns (uint256){\n        return number;\n    }\n}"
+           }
+       },
+       "settings": {
+           "optimizer": {
+               "enabled": false,
+               "runs": 200
+           },
+           "outputSelection": {
+               "*": {
+                   "": [
+                       "ast"
+                   ],
+                   "*": [
+                       "abi",
+                       "metadata",
+                       "devdoc",
+                       "userdoc",
+                       "storageLayout",
+                       "evm.legacyAssembly",
+                       "evm.bytecode",
+                       "evm.deployedBytecode",
+                       "evm.methodIdentifiers",
+                       "evm.gasEstimates",
+                       "evm.assembly"
+                   ]
+               }
+           },
+           "remappings": [],
+           "evmVersion": "paris"
+       }
+   }
+```
+
+* Enter a Contract name, this can be found in the Contract `Storage`. 
+* Select the compiler version, note to use the same compiler version used when compiling the contracts on Remix.
+
+At the bottom, below the form, click on the **verify** button and wait for the verification attempt to finish. If successfull, you’ll see a success screen.
+
+![Verify Contract - Rootstock Testnet Explorer](/img/developers/quickstart/14-remix.png)
+
+![Verify Contract Success - Rootstock Testnet Explorer](/img/developers/quickstart/15-remix.png)
+
+Visit [https://explorer.testnet.rootstock.io/address/0x1362ab9e72513807ce6c4c26250d7214ea98b783?__ctab=Code](https://explorer.testnet.rootstock.io/address/0x1362ab9e72513807ce6c4c26250d7214ea98b783?__ctab=Code), and you’ll see all the solidity code of the contract and its dependencies.
```

Comment:
I would just leave a screenshot showing that now the Contract Code tab now shows Verification data, avoiding that link. Since for the user will be a different address (also, if you plan to reuse the example images from past comment reviews, address will mismatch, generating confussion to the user)